feat: provide submit options for `mr create` and `issue create`

profclems opened the pull request:
Provide submit options to allow users either submit the MR via the CLI or continue in browser or cancel.

If 'preview in browser' is selected, it opens up a browser window with a prefilled title, description and
other mr or issue options.

Note on Prefilling issue with url query parameters: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32449479
and slash commands: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32550046

Resolves #329

profclems committed:
feat: provide submit options for `mr create` and `issue create`

Provide submit options to allow users either submit the MR via the CLI or continue in browser or cancel.

If 'preview in browser' is selected, it opens up a browser window with a prefilled title, description and
other mr or issue options.

Note on Prefilling issue with url query parameters: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32449479
and slash commands: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32550046

Resolves #329

maxice8 commented on commands/issue/create/issue_create.go:
```diff
@@ -179,7 +179,7 @@ func NewCmdCreate(f *cmdutils.Factory) *cobra.Command {
 	issueCreateCmd.Flags().StringVarP(&opts.Title, "title", "t", "", "Supply a title for issue")
 	issueCreateCmd.Flags().StringVarP(&opts.Description, "description", "d", "", "Supply a description for issue")
-	issueCreateCmd.Flags().StringVarP(&opts.Labels, "label", "l", "", "Add label by name. Multiple labels should be comma separated")
+	issueCreateCmd.Flags().StringSliceVarP(&opts.Labels, "label", "l", []string{}, "Add label by name. Multiple labels should be comma separated")
```
How it works on the command line? We call --label multiple times?

profclems replied:
`--label` can be called multiple times. It also accepts comma-separated values in a string and converts them to slice

profclems committed:
feat(cmd/issue/create): add submit options

profclems committed:
add ask stubber to test

maxice8 commented:
I assume this will be merged today for the release ?

profclems committed:
fix antipattern

https://deepsource.io/gh/profclems/glab/run/a58b2e14-4d0e-4a12-a735-c4ea1a8c685f/go/CRT-A0011

profclems commented:
> I assume this will be merged today for the release ?

Yeah sure

profclems commented:
Finally what I have been waiting for has been merged. I can do the release now https://github.com/ScoopInstaller/Main/pull/1640.
`glab` is now in the scoop main bucket
